migrations/educandu-2021-11-24-01-create-task-processing-collections.js

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line camelcase
+export default class Educandu_2021_11_24_01_create_task_processing_collections {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async up() {
+    await this.db.createCollection('batches');
+    await this.db.createCollection('batchLocks');
+    await this.db.createCollection('tasks');
+    await this.db.createCollection('taskLocks');
+
+    await this.db.collection('taskLocks').createIndexes([
+      {
+        name: '_idx_expires_',
+        key: { expires: 1 },
+        expireAfterSeconds: 0
+      }
```

Comment:
Should we maybe also have indices on
`tasks`: `taskType`, `batchId`, `processed`
`batches`: `batchType`, `completedOn`
?
Or you were thinking we will add them as we develop, later on?

Reply:
I think we should introduce them as needed (see my comment above) :-)

Reply:
Got it 😆 Ok then